kernel/src/main/java/org/kframework/kserver/KServerOptions.java

```diff
@@ -15,4 +15,8 @@
 
     @Parameter(names={"--port", "-p"}, description="The port to start the server on.")
     public int port = 2113;
+
+    @Parameter(names={"--socket"}, description="The socket to start the server on.")
```

Comment:
this should say something like "The directory to put the unix domain socket in"

Reply:
Done.